README.md

```diff
@@ -3,6 +3,13 @@
 The CSI-Addons project hosts extensions to the [CSI specification][csi_spec]
 that provide advanced storage operations.
 
+## Reclaim Space
+
+The [Reclaim Space](reclaimspace/README.md) specification sefines an extension
```

Comment:
@nixpanic sefines/defines.

Reply:
Thanks, that has been corrected now too.